src/grading.py

```diff
@@ -419,9 +460,11 @@ def EncodeImg(Ii,pathin,pathout):
 
 ############################################################################
 # écriture fichier out
-xmlout =  open(fileout, 'w')
-result_tree.write(xmlout, pretty_print=True,encoding="utf-8")
-xmlout.close()
+# xmlout =  open(fileout, 'w')
+# result_tree.write(xmlout, pretty_print=True,encoding="utf-8")
+# xmlout.close()
+result_tree.write(fileout, pretty_print=True,encoding="utf-8")
+print(result_tree)
```

Comment:
remove the print for production

Reply:
Done and other print (use deb option for some of it)

Reply:
done